src/inspectors/package/PHPPackageInspector.ts

```diff
@@ -0,0 +1,99 @@
+import { PackageInspectorBase } from './PackageInspectorBase';
+import { injectable, inject } from 'inversify';
+import { Types } from '../../types';
+import { keys } from 'lodash';
+import { DependencyType } from '../IPackageInspector';
+import { IFileInspector } from '../IFileInspector';
+
+@injectable()
+export class PHPPackageInspector extends PackageInspectorBase {
+  private fileInspector: IFileInspector;
+  private composerJson!: composerJSON;
+  private hasLockfileFile!: boolean;
+
+  constructor(@inject(Types.IFileInspector) fileInspector: IFileInspector) {
+    super();
+    this.fileInspector = fileInspector;
+  }
+
+  async init(): Promise<void> {
+    try {
+      this.debug('PHPPackageInspector init started');
+      const composerJsonString = await this.fileInspector.readFile('composer.json');
+      this.hasLockfileFile = await this.fileInspector.exists('composer.lock');
+      this.composerJson = JSON.parse(composerJsonString.replace(/\\/g, '/'));
+      this.packages = [];
+      this.addPackages(this.composerJson['require'], DependencyType.Runtime);
```

Comment:
Please, use it as `this.composerJson.require`. :)

Reply:
Addressing this in the next commit.